server.key

```diff
@@ -0,0 +1,15 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
The main game runs fine on ssl. I just have got the live reload to work. I may play with a dummy project to sort it out. Will keep you posted. Maybe we can create a grunt servessl option or something


> On 14 Feb 2015, at 2:46 am, Evan Rosson <notifications@github.com> wrote:
> 
> In server.key:
> 
> > @@ -0,0 +1,15 @@
> > +-----BEGIN RSA PRIVATE KEY-----
> I see - my mistake. Reverted the delete; these files exist again.
> 
> Did you have any luck running it as ssl?
> 
> —
> Reply to this email directly or view it on GitHub.
>